awx/main/models/inventory.py

```diff
@@ -1043,6 +1043,14 @@ class Meta:
         max_length=1024,
         blank=True,
         default='',
+        help_text=_('Relative path of inventory in source control.'),
+    )
+    plugin_path = models.CharField(
+        max_length=1024,
+        blank=True,
+        default='',
+        help_text=_('Inventory plugin to use for import, use leading @ to get '
```

Comment:
Why wouldn't it just always be relative? Why even bother with the `@`? Wouldn't that make sense as the default? We don't really want folks escaping that directory anyway do we?

Reply:
There are 2 things that need to be specified - the plugin code, and the name of the plugin(s) to be enabled. You could have the plugin in the search path, but not enable that plugin, this is how the Ansible core feature works. In this implementation, if you don't use the `@`, you are specifying the plugin name, and this could refer to a plugin shipped with Ansible core, or in a relative `inventory_plugins` directory.

@chrismeyersfsu offered what I think is the most workable suggestion yet, don't allow specifying the relative path at all, so if they want to use a plugin, it must be in folder `inventory_plugins` next to their inventory file.

Reply:
I think I like that one